core/src/main/scala/com.snowplowanalytics.stream/loader/transformers/eventTransformers/BadEventTransformer.scala

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
are this and other class we added or copied from some version of snowplow elastic loader r from our repo?

Reply:
They are from 0.12v of snowplow loader